### PR TITLE
Added support for intel and new examples

### PR DESCRIPTION
**Pull request by mauro-balades**

### CODE REVIEW

**Comment by mauro-balades** [comment]
Why does it include past commits? GitHub really has to do something about this.

File: lib/compilers/dotnet.ts [hunk 1]
```diff
@@ -324,10 +324,7 @@ class DotNetCompiler extends BaseCompiler {
                 if (envVar) {
                     const [name] = envVar.split('=');
                     const normalizedName = name.trim().toUpperCase();
-                    if (
-                        normalizedName === 'DOTNET_JITDISASM' ||
-                        normalizedName === 'DOTNET_JITDISASMASSEMBILES'
-                    ) {
+                    if (normalizedName === 'DOTNET_JITDISASM' || normalizedName === 'DOTNET_JITDISASMASSEMBILES') {
```
Comment (mattgodbolt) [inline lib/compilers/dotnet.ts:327]: Why did this seemingly unrelated file change too? Can you undo this change unless it's strictly needed?

File: lib/compilers/snowball.ts [hunk 2]
```diff
@@ -84,7 +84,7 @@ export class SnowballCompiler extends BaseCompiler {
                 options = options.concat('--emit', 'asm');
             }
             // TODO:
```
Comment (mattgodbolt) [inline lib/compilers/snowball.ts:86]: Please remove this TODO as it's now "done" I think?

**Review by mattgodbolt — CHANGES_REQUESTED** [review]

File: lib/compilers/dotnet.ts (hunk 1 above)
Comment (mattgodbolt) [reply lib/compilers/dotnet.ts:327]: I think this is our fault: some code got in that wasn't autoformatted... I'm going to fix this in main and then that will probably cause this to go away when you merge main! Sorry for the trouble.

Comment (mauro-balades) [reply lib/compilers/dotnet.ts:327]: What do you think I should do then? Ignore it?

**Review by mattgodbolt — APPROVED** [review]
